src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -982,11 +982,11 @@ protected final OpResult _put(DHTKey key, ByteBuffer value, StorageParametersAnd
         }
         
         if (debugVersion) {
-            System.out.println("StorageParameters: " + storageParams);
+            Log.fineAsync("StorageParameters: %s", storageParams);
         }
         if (debug) {
-            System.out.println("_put " + key + " " + value + " "+ storageParams.getChecksumType());
-            System.out.println(userData == null ? "null" : userData.length);
+            Log.fineAsyncf("_put %s %s %s", key, value, storageParams.getChecksumType());
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect